nautobot/extras/forms.py

```diff
@@ -807,12 +776,7 @@ class StatusFilterForm(BootstrapMixin, CustomFieldFilterForm):
 
     model = Status
     q = forms.CharField(required=False, label="Search")
-    content_types = MultipleContentTypeField(
-        queryset=ContentType.objects.filter(FeatureQuery("statuses").get_query()).order_by("app_label", "model"),
-        required=False,
-        label="Content type(s)",
-        widget=StaticSelect2Multiple(),
-    )
+    content_types = MultipleContentTypeField(feature="statuses", required=False, label="Content type(s)")
```

Comment:
Something isn't quite right w/ this change. I'm trying to figure out why.

![Add_a_new_status_-_Nautobot](https://user-images.githubusercontent.com/138052/112172184-b8277d00-8bb1-11eb-9a83-d675bb8c66f4.png)